datasets/README.md

```diff
@@ -1,14 +1,80 @@
 # Datasets Folder.
 
-## Purpose
-This folder is here to act as a designated placeholder for all the datasets this project uses.
-The reasoning for this is that the datasets are too large to have in the repo, and besides, the datasets won't be changing anyways.
-So, in order to maintain consistency across all developers and users, all datasets should be listed below with a download link and the name of the file that the program is expecting it to be called.
-No files in this folder will be tracked
+## Datasets
+### Income Dataset
+_2013 Tax Income Statistics_   
+Dataset Filename: `income-data.csv`    
+UID: ZIPCODE
 
+| Attribute    | Type | Description |
+| ---          | ---  | ---         |
+| STATEFIPS    | Char | 01-56       |
+| STATE        | Char | Two-digit State abbreviation code |
+| ZIPCODE      | Char | |
+| AGI_STUB     | Num  | See below |
+| NUM_RETURNS  | Num  |  |
```

Comment:
Amount of space in the empty description box is different for this and ZIPCODE. Not sure if it really matters though